src/redisearch_rs/encode_decode/tests/varint.rs

```diff
@@ -0,0 +1,262 @@
+use std::u128;
+
+use encode_decode::varint::*;
+
+#[test]
+fn test_u32() {
+    let values: [u32; 12] = [123456789, 987654321, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9];
+    let expected_lens = [4, 5, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1];
+
+    for (i, value) in values.iter().enumerate() {
```

Comment:
```suggestion
    for (i, value) in values.into_iter().enumerate() {
```
You don't need to access `values` again so can just take ownership of it. It will also making accessing the `value`s cleaner

Reply:
Switched over in [41270e9](https://github.com/RediSearch/RediSearch/pull/6346/commits/41270e9f87fa29063c51847c97c2eea8b42dc1b2)